schemas/json/examples/ReferencesJsonExample.json

```diff
@@ -0,0 +1,34 @@
+{
+  "submodels":[
+    {
+      "modelType":{
+        "name":"Submodel"
+      },
+      "idShort":"S1",
+      "identification":{
+        "id":"http://env.com/submodels/S1",
+        "idType":"IRI"
+      },
+      "submodelElements":[]
+    }
+  ],
+  "conceptDescriptions":[],
+  "assetAdministrationShells":[
+    {
+      "modelType":"AssetAdministrationShell",
+      "submodels":[
+        {
+          "keys":[
+            {
+              "idType":"IRI",
+              "local":true,
```

Comment:
no local in Key since V3.0RC01

Reply:
Table 15 "Exemplary minimal JSON for References" shows this example in the published version of V3.0RC01

Reply:
I see... so this is a bug in V3.0RC01 that we need to fix in V3.0RC02 and we should also in github for V3.0RC01

Reply:
After all, this is the initial commit. It should be a direct copy from the book "AAS in Details". Every bugfixes and change should be in a separated commit after initial one. Otherwise, it's unclear where, when and why we diverge from the original.